staging/src/k8s.io/client-go/tools/watch/retrywatcher.go

```diff
@@ -190,7 +189,7 @@ func (rw *RetryWatcher) doReceive() (bool, time.Duration) {
 				errObject := apierrors.FromObject(event.Object)
 				statusErr, ok := errObject.(*apierrors.StatusError)
 				if !ok {
-					klog.Error(fmt.Sprintf("Received an error which is not *metav1.Status but %s", dump.Pretty(event.Object)))
+					klog.ErrorS(errObject, "Received an error which is not *metav1.Status", "eventObject", dump.Pretty(event.Object))
```

Comment:
So `event.Object` was already getting dumped. I'm still wondering whether we know what it contains...

Reply:
In the context of structured logging, formatting a complex type should be left to the logging backend. In other words:
```suggestion
					klog.ErrorS(errObject, "Received an error which is not *metav1.Status", "eventObject", event.Object)
```